### ISSUE TITLE
xtests currently fail

### ISSUE THREAD

**Issue by kballard** [ogham/exa#165]
The very first test in the xtests suite fails (and probably many others too). It looks like the issue is the files have a timestamp from 2016, and the xtest output files all expect `exa` to spit out a time, but since the year is now 2017, `exa` prints the year instead of the time, causing the tests to fail.

The trivial fix here, changing the timestamp to 2017, would of course break again in 2018. Would it be possible to change the clock of the Vagrant machine so it thinks the time is 2016, or would that cause issues somehow? Alternatively, we could just change the test output to expect 2016, but I don't know if you particularly care about having the xtests test the timestamp display.

**Comment by kballard** [comment]
Looks like you might be able to use [libfaketime](https://github.com/wolfcw/libfaketime) to make `exa` think the current time is sometime in 2016 without actually changing the system clock.

**Comment by ogham** [comment]
I’m _so_ glad you reported this. Last night I provisioned the exa VM on a different machine, and none of the tests passed, and if I didn’t remember seeing this a couple of hours before I would have spent ages going round in circles trying to find what was wrong!

Turns out, the test cases on my own computer hadn’t actually been updated since last year. I should really provision more to prevent being caught out by this in the future.

I did look into libfaketime, but the “fix” I plan on committing involves dynamically getting the current year in the Vagrantfile (it’s Ruby code, after all), and putting a check in the tests runner to see if the year has changed.
